### PR TITLE
Improve VPN notification UX

### PR DESCRIPTION
**Pull request by HiddenCoderIR**
This PR includes enhancements to the VPN notification:

1. Add disconnect button
2. Customize notification text 
3. Use dedicated notification icon

The disconnect button allows users to easily disconnect the VPN directly from the notification without entering the app.

Customizing the text provides more transparency into which mode the user has enabled. For example, "Oblivion - Psiphon (United States)".

A notification-specific icon is added to fix the issue of the launcher icon not displaying properly in the notification.

Together these improve the notification UX by adding convenient controls and readable status.

------------

Before:

Screenshot_20240220_235459

After:

Screenshot_20240221_000322

### CODE REVIEW

File: app/src/main/java/org/bepass/oblivion/OblivionVpnService.java [hunk 1]
```diff
@@ -430,26 +430,50 @@ public void setLastKnownState(ConnectionState lastKnownState) {
...
 
+    private String getNotificationText() {
+        if(fileManager.getBoolean("USERSETTING_psiphon")) {
+            String countryCode = fileManager.getString("USERSETTING_country");
+            String countryName;
+            if("".equals(countryCode)) {
+                countryName = "Automatic";
+            } else {
+                countryName = CountryUtils.getCountryName(countryCode);
+            }
```
Comment (markpash) [inline app/src/main/java/org/bepass/oblivion/OblivionVpnService.java:441]: Something like this looks a little cleaner:
```java
String countryName = "Automatic";
if (!countryCode.equals("")) {
  countryName = CountryUtils.getCountryName(countryCode);
}
```

**Review by markpash — CHANGES_REQUESTED** [review]
Thanks for the great addition! Just one tiny comment on the code style.
Also please rebase your work on `main` as I think a lot of this country related code has changed now and so this PR might need to adapt.

File: app/src/main/java/org/bepass/oblivion/OblivionVpnService.java (hunk 1 above)
Comment (HiddenCoderIR) [reply app/src/main/java/org/bepass/oblivion/OblivionVpnService.java]: Done!

**Comment by markpash** [comment]
Great! This looks good! Can you squash this into one commit and rebase cleanly on `main`? There's a merge commit in your PR which is confusing.

**Comment by markpash** [comment]
nvm, I pushed to your branch and fixed it :)
merging!

**Pull request merged**